feature/pokemonList/src/main/java/com/matheus/mendes/pokedex/pokemonlist/presentation/PokemonListViewModel.kt

```diff
@@ -10,8 +10,9 @@ import kotlinx.coroutines.flow.onCompletion
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.flow.onStart
 import kotlinx.coroutines.launch
+import retrofit2.HttpException
 
-
+const val ERROR = "Não foi possível conectar a pokedex"
```

Comment:
não é muito interessante deixar mensagens pro usuario em constantes, caso vc precise traduzir o app essa mensagem nao seria traduzida, apenas os textos que ficam no strings

Reply:
talvez vc poderia criar uma exception e fazer um when para quando for exception X vc busca string x do arquivo de strings